airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -31,15 +32,39 @@ class SnowflakeSqlOperations extends JdbcSqlOperations implements SqlOperations
   private static final String NO_PRIVILEGES_ERROR_MESSAGE = "but current role has no privileges on it";
   private static final String IP_NOT_IN_WHITE_LIST_ERR_MSG = "not allowed to access Snowflake";
 
+  private final boolean use1s1t;
+
+  public SnowflakeSqlOperations() {
+    this.use1s1t = TypingAndDedupingFlag.isDestinationV2();
+  }
+
   @Override
   public String createTableQuery(final JdbcDatabase database, final String schemaName, final String tableName) {
-    return String.format(
-        "CREATE TABLE IF NOT EXISTS %s.%s ( \n"
-            + "%s VARCHAR PRIMARY KEY,\n"
-            + "%s VARIANT,\n"
-            + "%s TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp()\n"
-            + ") data_retention_time_in_days = 0;",
-        schemaName, tableName, JavaBaseConstants.COLUMN_NAME_AB_ID, JavaBaseConstants.COLUMN_NAME_DATA, JavaBaseConstants.COLUMN_NAME_EMITTED_AT);
+    if (use1s1t) {
+      return String.format(
+          """
+              CREATE TABLE IF NOT EXISTS %s.%s (
+                "%s" VARCHAR PRIMARY KEY,
+                "%s" TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp(),
+                "%s" TIMESTAMP WITH TIME ZONE DEFAULT current_timestamp(),
```

Comment:
I think the default for LOADED_AT should be null (it's null until T&D happens).

Reply:
this is how you know I copypasted this without even reading it :P